browser-tests/pages/loginSelector.ts

```diff
@@ -0,0 +1,11 @@
+import { Selector } from 'testcafe';
+export const loginSelector = {
+  submitButton: Selector('button[type="submit"]'),
+  errorText: Selector('span[class="hds-text-input__helper-text"]'),
+  day: Selector('input[id="ym-date-input-date-id"]'),
+  month: Selector('input[id="ym-date-input-month-id"]'),
+  year: Selector('input[id="ym-date-input-year-id"]'),
+  paperForm: Selector(
+    'a[href="https://palvelukartta.hel.fi/fi/search?q=nuorisotalo"]'
+  ),
```

Comment:
Some of these selectors seem relatively brittle to me. I looked through some examples, for instance berth control, and they used a similar approach. I guess that something in TestCafe makes this approach palatable?

Reply:
These seemed little iffy to me too. I'll replace this with `test-id` attribute.